### PR TITLE
show docker-py test failures

### PR DESCRIPTION
**Pull request by rade**

### CODE REVIEW

**Comment by rade** [comment]
NB: I have branched this off 1.0 since it's rather important we know why failures occur there.

File: test/600_proxy_docker_py_test.sh [hunk 1]
```diff
@@ -4,13 +4,19 @@
...
-  -v /var/run/docker.sock:/var/run/docker.sock \
-  joffrey/docker-py python tests/integration_test.py"
+
+if docker_on $HOST1 run \
+    -e NOT_ON_HOST=true \
+    -e DOCKER_HOST=tcp://172.17.42.1:12375 \
+    -v /tmp:/tmp \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    joffrey/docker-py python tests/integration_test.py ; then
+    assert_raises "true" 0
```
Comment (paulbellamy) [inline test/600_proxy_docker_py_test.sh]: unneeded `0`

Comment (rade) [inline test/600_proxy_docker_py_test.sh]: fixed

**Comment by paulbellamy** [comment]
LGTM

**Pull request merged**